src/meshapi/views/map.py

```diff
@@ -122,7 +122,7 @@ def list(self, request: Request, *args: List[Any], **kwargs: Dict[str, Any]) ->
             ap_json = {
                 # Hacky, but we have no choice, we need this to present as a "node" object to the
                 # map frontend and not conflict with any existing installs
-                "id": 1_000_000 + ap.id,
+                "id": 1_000_000 + (ap.id.int % 1_000_000),
```

Comment:
this is compatibility with the old map. That's fine, I guess lol

Reply:
🫡